schedule/yast/sle/flows/default_s390x_kvm.yaml

```diff
@@ -39,8 +39,7 @@ installation:
   - installation/launch_installation
   - installation/confirm_installation
   - installation/performing_installation/perform_installation
-stop_timeout_system_reboot:
-  - installation/performing_installation/stop_timeout_system_reboot_now
+stop_timeout_system_reboot: []
```

Comment:
you cannot do that without breaking many test suite, please coordinate with @lemon-suse he is working in similar solution, which is create another yaml default for textmode I think.

Reply:
but why don't use `schedule/yast/sle/flows/default_s390x_kvm_textmode.yaml`? it's still [**works**](https://openqa.suse.de/tests/14759341#settings) and we don't need `system_preparation` in PR #19622

Reply:
Reuse the schedule/yast/sle/flows/default_s390x_kvm_textmode.yaml now. For system_preparation, not a must to remove it, and to unify schedule of test suite, we can't simply to overwrite that in test suite yaml.

Reply:
@lemon-suse  @jknphy  restore it to it's original and please take a look again.  Thanks!